Misc/NEWS.d/next/Documentation/2022-10-19-07-15-52.gh-issue-98366.UskMXF.rst

```diff
@@ -0,0 +1 @@
+Use sphinx.locale._ as the gettext function in pyspecific.py.
```

Comment:
Does the NEWS is required? or should I add the `skip news` label to this PR?

Reply:
ok thanks! I've dropped the commit.

Reply:
Cool!  For future Python PRs, please don’t use force pushes, github has a poor experience for reviewers.
Guide: https://devguide.python.org/getting-started/pull-request-lifecycle/index.html